docs/notebooks/feasible_sets.pct.py

```diff
@@ -230,6 +240,77 @@ def objective_function(x):
 )
 
 
+# %% [markdown]
+# ## Active learning with Integrated Variance Reduction acquisition function
+#
+# An alternative to the `ExpectedFeasibility` function is called `IntegratedVarianceReduction`. This acquisition has the advantage of allowing batches, as we illustrate below. It is more expensive to compute than `ExpectedFeasibility`, as it writes as an integral over a set of input locations.
+
+# %%
+from trieste.acquisition.function import IntegratedVarianceReduction
+
+# Choose integration points uniformly over the design space
+integration_points = search_space.sample_halton(1000)
+acq_ivr = IntegratedVarianceReduction(
+    integration_points=integration_points,
+    threshold=[
```

Comment:
having to use a list even for a single threshold is suboptimal from a user perspective, do you mind allowing for both a scalar and a list?